frontend/README.md

```diff
@@ -46,6 +46,23 @@ For example, to only run the design system storybook:
 yarn dev -- --filter=@code-dot-org/design-system
 ```
 
+### Visual Snapshot Testing (Eyes)
+
+The design system uses [Applitools Eyes](https://applitools.com/platform/eyes/) via their [storybook integration](https://applitools.com/tutorials/sdks/storybook) to take a visual snapshot of a storybook component and
+compare it with baselines. Visual snapshots on pull requests and during the CI build.
+
+To run visual snapshots locally, first obtain an [Applitools API Key](https://applitools.com/docs/topics/overview/obtain-api-key.html).
+
+If using devcontainers, add the key to `.devcontainer/frontend/.env`. For everywhere else, assign the API key to the environment variable `APPLITOOLS_API_KEY`.
```

Comment:
No, `locals.yml` is not read for environment loading here as these tools use the dotenv framework. Rather it's injected via environment variables using the [dotenv](https://www.dotenv.org/) standard. (This is also done for Code.org via the `CDO_*` environment variables)

Reply:
Ok -- is there a reason not to include .env.example within the `frontend` directory to support native development (rather than having to set env variables manually)?

Reply:
Great idea, I actually had not considered this and put it in `.devcontainer/frontend` but it does make sense to put it in `frontend` instead because not everyone will use devcontainers. I went ahead and moved it over!

Reply:
Nice! Can you update the readme to cover that change?

Reply:
I updated it and also went back and double checked if things worked and looks like I missed the `dotenv` loader so it wasn't injecting the environment variable. I updated the PR to include the README and the `dotenv` loader as well.

Reply:
Awesome, thanks!